src/components/sound-card/sound-card.jsx

```diff
@@ -0,0 +1,31 @@
+import { useSounds } from "../../hooks/use-sounds";
+import purpleGoblin from "../../images/purple-goblin.png";
+
+export function SoundCard({ soundInfo }) {
+  const card = {
+    image: soundInfo?.photo,
+    title: soundInfo?.title,
+    audio: soundInfo?.audio,
+  };
+
+  const { playSound } = useSounds();
+
+  function handleSoundClick() {
+    console.log(card.audio);
+    playSound("https://" + card.audio);
```

Comment:
Can we make the `https://` prefix optional depending on if the `audio` string already starts with that or not? It'll depend on how we seed the database (Fred's work), but would be nice to support the audio link either way. Also, as a nit, we can use template literals here (leverage backticks) instead of string concatenation.